src/resources/cms/types.md

```diff
@@ -0,0 +1 @@
+# CMS Types
```

Comment:
Do you need this as you describe it in the CMS docs

Reply:
Think I will move it from CMS Docs and place it here as I think Types will go through a bit more embellishment and maybe some diagrams in the next iteration